components/gc-stp-stp/gc-stp-stp-en.html

```diff
@@ -0,0 +1,63 @@
+---
+{
+	"title": "[Service name] - 2. [Step / section page name]",
+	"secondTitle": "2. [Step / section page name]",
+	"language": "en",
+	"altLangPage": "gc-stp-stp-fr.html",
+	"breadcrumbs": [
+		{ "title": "[Institution name]", "link": "#" },
+		{ "title": "[Topic name]", "link": "#" },
+		{ "title": "[Service name]", "link": "#" }
+	],
+	"secondlevel": false,
+	"dateModified": "2023-06-28",
+	"share": "true"
+}
+---
+<p class="gc-byline"><strong>From <a href="../institutional/institution-en.html">[Institution name]</a></strong></p>
+
+<div class="gc-stp-stp">
+	<div class="row">
+		<ul class="toc lst-spcd col-md-12">
+			<li class="col-md-4 col-sm-6"><a class="list-group-item" href="index-en.html">1. [Step / section page name]</a></li>
+			<li class="col-md-4 col-sm-6"><a class="list-group-item active">2. [Step / section page name]</a></li>
+			<li class="col-md-4 col-sm-6 clr-lft-sm"><a class="list-group-item" href="page3-en.html">3. [Step / section page name]</a></li>
+			<li class="col-md-4 col-sm-6 clr-lft-md clr-lft-lg"><a class="list-group-item" href="page4-en.html">4. [Step / section page name]</a></li>
+			<li class="col-md-4 col-sm-6 clr-lft-sm"><a class="list-group-item" href="page5-en.html">5. [Step / section page name]</a></li>
+			<li class="col-md-4 col-sm-6"><a class="list-group-item" href="page6-en.html">6. [Step / section page name]</a></li>
+		</ul>
+	</div>
+</div>
```

Comment:
This don't like the version 2 of the component.

This is not a "toc" (table of content) if we compare on how the "toc" CSS is currently used for the in-page table of content component.